quickstarts/kubernetes/nginx/config.yml

```diff
@@ -0,0 +1,51 @@
+name: NGINX Ingress Controller
```

Comment:
This just needs to called `slug` and be hyphenated lowercase
```suggestion
slug: nginx-ingress-controller
```

Reply:
Oops, I'll get that fixed!